src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderViewMenu.vue

```diff
@@ -57,6 +61,16 @@ export default {
 				label: this.$t('calendar', 'Month'),
 			}]
 		},
+		shortKeyConf() {
+			return {
+				timeGridDay: ['d'],
+				timeGridDay_Num: [1],
```

Comment:
Why different keys for numbers (`timeGridDay` vs `timeGridDay_Num`) ?

Reply:
Because we need two different keys for the object. If we put `d` and `1` into one array, it wouldn't be `d` or `1` but combination of `d` and `1`